### DISCUSSION TITLE
Sun Tracker

### DISCUSSION THREAD

**Discussion by mpvm** [arendst/Tasmota#21566] [Ideas]
It requires multiple light sensor which can work on Sun shine, please change drivers code for xsns_16_tsl2561.ino and xsns_41_max44009.ino in a similar fashion as with the BH1750 or INA226

Please add support for multiple instance same sensor if sensor hat more than one i2c possible address to select
Thanks

**Comment by sfromis** [comment]
As mentioned on Discord, the simplest current approach would be to use analog LDR sensors on ESP32, where you can have multiple. While the digital I2C ones are generally better for measuring light level in lux, comparing for sun tracking doesn't require much.
